### ISSUE TITLE
How to reload calendar from existing instance?

### ISSUE THREAD

**Issue by spizzico7** [PeteC/DSLCalendarView#25]
i have set self.calendarView.delegate = self; in my UIViewController, i want to reload only calendar without re-instantiate all calendar, how can i do?

**Comment by rahulshirphule** [comment]
Yes guys I also want the same to reload data after first initialisation Please help...! Thanks
